repository/rabbitmq/operator/operator.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kudo.dev/v1beta1
+name: "rabbitmq-operator"
```

Comment:
let's use `rabbitmq` name as it's implicit that it will be an KUDO operator
and install cmd will look like `kubectl kudo install rabbitmq` vs `kubectl kudo install rabbitmq-operator`